### ISSUE TITLE
CPT was not updated to latest version in gcc8 image

### ISSUE THREAD

**Issue by grisumbras** [conan-io/conan-docker-tools#138]
### Description of Problem, Request, or Question
gcc8 image doesn't have conan_package_tools updated to latest version (0.29.0).

### Environment Details
* Conan Docker Tools image: **conanio/gcc8:1.17.0**
* Python version (If you are building some image): **python 3.7.0**

### Steps to reproduce (Include if Applicable)
```
docker run -it --rm conanio/gcc8:1.17.0
pip show conan_package_tools
```

### Build logs (Include if Available)
```
To run a command as administrator (user "root"), use "sudo <command>".
See "man sudo_root" for details.

conan@2b2e82b8f44f:~$ pip show conan_package_tools
Name: conan-package-tools
Version: 0.28.1
Summary: Packaging tools for Conan C/C++ package manager
Home-page: https://github.com/conan-io/conan-package-tools
Author: JFrog LTD. Luis Martinez de Bartolome
Author-email: luism@jfrog.com
License: MIT
Location: /opt/pyenv/versions/3.7.1/lib/python3.7/site-packages
Requires: six, conan, tabulate
Required-by:
conan@2b2e82b8f44f:~$
```

**Comment by uilianries** [comment]
Some Docker builds failed due connection error. I just restarted now.

**Comment by uilianries** [comment]
@grisumbras Done, a new image is available with CPT 0.29.0

**Comment by grisumbras** [comment]
Thanks for the quick response!
